### PR TITLE
ref(ui): Codesplit project dashboard, remove unused files

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/routes.jsx [hunk 1]
```diff
@@ -879,7 +878,12 @@ function routes() {
           <Route path="issues/" component={errorHandler(Stream)} />
 
           <Route path="searches/:searchId/" component={errorHandler(Stream)} />
-          <Route path="dashboard/" component={errorHandler(ProjectDashboard)} />
+          <Route
+            path="dashboard/"
+            componentPromise={() =>
+              import(/*webpackChunkName:"ProjectDashboard"*/ './views/projectDashboard')}
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/routes.jsx]: Please add spaces like #11348

File: src/sentry/static/sentry/less/sentry.less [hunk 2]
```diff
@@ -34,7 +34,6 @@
 
 @import url('./admin.less');
 @import url('./auth.less');
-@import url('./dashboard.less');
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/less/sentry.less]: ❤️

File: src/sentry/static/sentry/app/routes.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: Is that interpreted differently?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/routes.jsx]: No, but I'm just trying to make them consistent.

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**